src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -35,6 +35,7 @@
 
 // Enable detailed mDNS logging for received queries
 #undef DETAIL_LOGGING
+#define DETAIL_LOGGING
```

Comment:
Is this accidental? Should this be commented out - or just standard logging?

Reply:
Now has been changed to this:

```
#undef DETAIL_LOGGING
// #define DETAIL_LOGGING
```

What is the point of the second commented line?  This is what was asked for, but doesn't make any sense to me.

Reply:
Presuming this is OK since @woody-apple approved.